connect_ext_ppr/services/cbc_hub.py

```diff
@@ -0,0 +1,74 @@
+from connect_ext_ppr.client import CBCClient
+from connect_ext_ppr.models.cbc_extenstion import HubCredential
+
+
+class CBCService:
+
+    PLM_TYPE = 'http://com.odin.platform/inhouse-products/application'
+    SUBSCRIPTION_TYPE = 'http://parallels.com/aps/types/pa/subscription'
+
+    def __init__(self, hub_credential: HubCredential, verify_certificate: bool = False):
+        assert hub_credential, 'HubCredential can not be None!'
```

Comment:
Please, do not use assertions to check the required properties :-(

Reply:
Why not?